pymove/core/dataframe.py

```diff
@@ -3233,107 +3172,119 @@ def groupby(self, by=None, **kwargs):
-        # Returns a subset of the dask DataFrame columns based on the column dtypes.
+        """ Returns a subset of the dask DataFrame columns based on the column dtypes. """
+
         raise NotImplementedError("To be implemented")
 
     def astype(self):
-        # Casts a dask object to a specified dtype.
+        """ Casts a dask object to a specified dtype. """
+
         raise NotImplementedError("To be implemented")
 
     def sort_values(self):
-        # Sorts the values of the dask DataFrame.
+        """ Sorts the values of the dask DataFrame. """
+
         raise NotImplementedError("To be implemented")
 
     def reset_index(self):
-        # Resets the dask DataFrame's index, and use the default one.
+        """ Resets the dask DataFrame's index, and use the default one. """
+
         raise NotImplementedError("To be implemented")
 
     def set_index(self):
-        # Set the dask DataFrame index (row labels) using one or more existing columns or arrays (of the correct length).
+        """ Set the dask DataFrame index (row labels) using one or more existing columns or arrays (of the correct length). """
+
         raise NotImplementedError("To be implemented")
 
     def drop(self):
-        # Drops specified rows or columns of the dask Dataframe.
+        """ Drops specified rows or columns of the dask Dataframe. """
+
         raise NotImplementedError("To be implemented")
 
     def duplicated(self):
-        # Returns boolean Series denoting duplicate rows, optionally only considering certain columns.
+        """ Returns boolean Series denoting duplicate rows, optionally only considering certain columns. """
+
         raise NotImplementedError("To be implemented")
 
     def drop_duplicates(self):
-        # Removes duplicated rows from the data.
+        """ Removes duplicated rows from the data. """
+
         raise NotImplementedError("To be implemented")
 
     def shift(self):
-        # Shifts dask dataFrame index by desired number of periods with an optional time freq.
+        """ Shifts dask dataFrame index by desired number of periods with an optional time freq. """
+
         raise NotImplementedError("To be implemented")
 
     def all(self):
-        # Inidicates if all elements are True, potentially over an axis.
-        # Returns True unless there at least one element within the Dataframe axis that is False or equivalent.
+        """ Indicates if all elements are True, potentially over an axis. """
+
         raise NotImplementedError("To be implemented")
 
     def any(self):
-        # Inidicates if any element is True, potentially over an axis.
-        # Returns False unless there at least one element within the dask Dataframe axis that is True or equivalent.
+        """ Indicates if any element is True, potentially over an axis. """
+
         raise NotImplementedError("To be implemented")
 
     def isna(self):
-        # Detect missing values
+        """ Detect missing values """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 's') (D415)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5536634)